Jenkinsfile

```diff
@@ -27,10 +27,10 @@ pipeline {
                 '''
                 script{
                     String s = env.JOB_NAME
-                    s = s.substring(s.indexOf("/") + 1)
-                    s = s.substring(0, s.indexOf("/"));
-                    println(s);
-                    switch(s){
+                    String[] elements = s.split("/")
+                    String job_name = elements[-2]
```

Comment:
Are there always at least two elements? i.e. is there necessarily a `/` in `JOB_NAME` ?